packages/vuetify/src/components/VSlider/VSlider.ts

```diff
@@ -388,13 +388,12 @@ export default mixins<options &
         attrs: {
           role: 'slider',
           tabindex: this.isDisabled ? -1 : this.$attrs.tabindex ? this.$attrs.tabindex : 0,
-          'aria-label': this.label,
+          'aria-label': this.$attrs['aria-label'] || this.label,
           'aria-valuemin': this.min,
           'aria-valuemax': this.max,
           'aria-valuenow': this.internalValue,
           'aria-readonly': String(this.isReadonly),
           'aria-orientation': this.vertical ? 'vertical' : 'horizontal',
-          ...this.$attrs,
```

Comment:
Looks like the only purpose of non-prop attributes inherit here is for making sure `aria-label` takes highest precedence over `label` prop.

In addition, with this change applied, all events from ThumbContainer are consistent